### PR TITLE
feat: image and input

### PR DESCRIPTION
**Pull request by SomeEarth86**

### CODE REVIEW

File: Components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                    <code>
+&lt;div class="img-box"&gt;
+    &lt;img class="round-img" src="https://rb.gy/pbahrk" alt="goku-img"&gt;
+&lt;/div&gt;
+                    </code>
+                </pre>
+            </div>
+
+            <hr class="dashed">
+
```
Comment (vishknew) [inline Components/image/image.html:113]: you may remove unwanted space

Comment (SomeEarth86) [reply Components/image/image.html:113]: sure, intention was  to make clean code. Will do this in final iteration.

File: Components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            </div>
+
+            <hr class="dashed">
+
+
+        </main>
+
+    </div>
+
+
```
Comment (vishknew) [inline Components/input/input.html:166]: consider removing unwanted space at mulitple places.

**Review by VishnuVNairCodes — APPROVED** [review]
Except for unwanted spaces in code, looks good to merge.

File: Components/input/input.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+                    </div>
+
+                    <div class="primary-inp">
+                        <label >Enter password</label>
+                        <input type="password">
+                    </div>
+
+                    <div class="primary-inp">
+                        <label >Enter text</label>
+                        <textarea name="" id="" cols="30" rows="5"></textarea>
```
Comment (avindhawale) [inline Components/input/input.html:92]: keeping id attribute blank will lead to accessibility issue. Provide the values to id and name attribute or remove them if not require.

**Pull request merged**